src/packet_analysis/Manager.h

```diff
@@ -113,11 +132,21 @@ class Manager : public plugin::ComponentManager<Tag, Component> {
 	 */
 	AnalyzerPtr InstantiateAnalyzer(const std::string& name);
 
+	bool PermitUnknownProtocol(const std::string& analyzer, uint32_t protocol);
```

Comment:
I would rather call that `IsUnknownProtocolPermitted` but I see that this aligns to the naming for Weird.